docs/contributor_guide.rst

```diff
@@ -157,7 +157,9 @@ Testing
 Tests should be added to ``taichi/tests``.
 Use ``ti test`` to run all the tests.
 Use ``ti test -v`` for verbose outputs.
-Use ``ti test <filename(s)>`` to run specific tests. Feel free to omit the ``test_`` prefix and ``.py`` suffix in the filenames.
+Use ``ti test <filename(s)>`` to run specific tests. e.g. ``ti test numpy_io`` and ``ti test test_numpy_io.py`` are equivalent.
+Use ``ti test -a <arch(s)>`` for test against specified architectures. e.g. ``ti test -a opengl`` or ``ti test numpy_io -a cuda,metal``.
+Use ``ti test -c`` to run only the C++ tests. e.g. ``ti test -c alg_simp``
```

Comment:
This should be `ti -c test alg_simp`.

Reply:
Fixed `argparse` happiness by poping `argv[1]`. Now it's `ti test -c alg_simp` legal now.